yt/utilities/exceptions.py

```diff
@@ -742,3 +742,17 @@ def __str__(self):
         msg += "or:\n"
         msg += "  pip install %s\n" % self.module
         return msg
+
+class YTInvalidArgumentType(YTException):
```

Comment:
I'll let others weigh in on this because it seems more like an issue of convention, but why isn't this just (depending on the specific error) a `TypeError` or a `ValueError`? If you're going to dress it up, at least include some specific information, like what argument was the incorrect type/value.

Reply:
updated